Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -161,3 +209,89 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
         }
     }
 }
+
+// MARK: Dictionary Colon Rule
+extension ColonRule {
+
+    public func validateFile(_ file: File, kind: SwiftExpressionKind,
```

Comment:
but a comment at the call site does... ugh. anyway, no great solutions to any of this. do what you think is best.